Mage/src/main/java/mage/game/tournament/TournamentImpl.java

```diff
@@ -414,6 +414,10 @@ protected void openBoosters() {
                     player.getDeck().getCards().addAll(JumpstartPoolGenerator.generatePool(options.getLimitedOptions().jumpstartPacks));
                 }
             } else {
+                // each player gets their own fresh box so pools aren't negatively correlated
+                for (ExpansionSet set : sets) {
+                    set.openBoosterBox();
+                }
```

Comment:
like for example here, this code doesn't actually do anything

Reply:
yes it does